Content.Shared/Xenoarchaeology/Equipment/Components/ArtifactAnalyzerComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Xenoarchaeology.Equipment.Components;
+
+/// <summary>
+/// A machine that is combined and linked to the <see cref="AnalysisConsoleComponent"/>
+/// in order to analyze artifacts and extract points.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
+public sealed partial class ArtifactAnalyzerComponent : Component
+{
+    /// <summary>
+    /// How long it takes to analyze an artifact
+    /// </summary>
+    [DataField]
+    public TimeSpan AnalysisDuration = TimeSpan.FromSeconds(30);
```

Comment:
unused
maybe scan artifact before displaying grid?

Reply:
well... is a thing to consider. Maybe leave for later? Implementing now seems like scope creep. Remove it  and ReadyToPrint flag for now?